wallet-new/src/Cardano/Wallet/API/V1/Migration/Types.hs

```diff
@@ -99,7 +99,7 @@ instance Migrate (OldStorage.SyncStatistics, Maybe Core.ChainDifficulty) V1.Sync
                                                  (calculateEstimatedRemainingTime wspThroughput)
                                                  remainingBlocks))
                            <*> pure (tput wspThroughput)
-                           <*> pure (V1.mkSyncPercentage (round @Double $ percentage))
+                           <*> pure (V1.mkSyncPercentage (floor @Double $ percentage))
```

Comment:
As I was at it, I seized the chance to do a small UX improvement: using `floor` here means that `99....` won't be rounded to `100` but rightly to `99`, so that users won't be confused on why syncing is at 100% but they still cannot operate the wallet.

Reply:
:+1: